app/webpacker/styles/govspeak/_details.scss

```diff
@@ -0,0 +1,5 @@
+.gem-c-govspeak {
+  .govuk-details {
+    padding-top: 5px;
+  }
+}
```

Comment:
What does this do? (Or rather, why is it here? :D )

Reply:
It just adds a tiny bit of space after a video for the details section. It's just to make it look a bit nicer. I'll add an image.

Reply:
I mentioned it to Dom and she just said if have somewhere to add it in then add a bit of spacing